bitnami/cilium/Chart.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+annotations:
+  category: Infrastructure
+  licenses: Apache-2.0
+  images: |
+    - name: cilium
+      image: docker.io/bitnami/cilium:1.15.5-debian-12-r6
+    - name: cilium-operator
+      image: docker.io/bitnami/cilium-operator:1.15.5-debian-12-r1
+apiVersion: v2
+appVersion: 1.15.4
+dependencies:
+  - condition: etcd.enabled
+    name: etcd
+    repository: oci://registry-1.docker.io/bitnamicharts
+    tags:
+      - cilium-database
+    version: 10.x.x
+  - name: common
+    repository: oci://registry-1.docker.io/bitnamicharts
+    tags:
+      - bitnami-common
+    version: 2.x.x
+description: Cilium is an eBPF-based networking, observability, and security for Linux container management platforms like Docker and Kubernetes.
+home: https://bitnami.com
+icon: https://bitnami.com/assets/stacks/cilium/img/cilium-stack-220x234.png
+keywords:
+  - cilium
+  - cni
+  - networking
+  - observability
+  - security
+maintainers:
+  - name: Broadcom, Inc. All Rights Reserved.
+    url: https://github.com/bitnami/charts
+name: cilium
+sources:
+  - https://github.com/bitnami/containers/tree/main/bitnami/cilium
```

Comment:
Missing the chart source folder, this is essential for automation tools to fetch the changelog.

Reply:
done at https://github.com/bitnami/charts/pull/26374/commits/129afd439cbffb07203c84e7db2880f82caa370f